packages/notifi-react-example/src/NotifiCard/NotifiCard.tsx

```diff
@@ -90,13 +95,165 @@ export const NotifiCard: React.FC = () => {
             },
           }}
         />
+        NotifiIntercomCard
         <NotifiIntercomCard
           darkMode
           inputLabels={inputLabels}
           inputSeparators={intercomInputSeparators}
           cardId="1045f61752b148eabab0403c08cd60b2"
         />
       </NotifiContext>
+
+      <NotifiContext
+        dappAddress=""
+        env="Development"
+        walletBlockchain="SOLANA"
+        walletPublicKey="string"
+        signMessage={async (msg: Uint8Array) => msg}
+        hardwareLoginPlugin={{ sendMessage: async (msg: string) => msg }}
+      >
+        Dummy Demo NotifiSubscriptionCard: Preview page
+        <NotifiSubscriptionCard
+          darkMode
+          cardId=""
+          demoByJSONConfig={{ view: 'preview', data: JSON.parse(data) }}
+        />
+        Dummy Demo NotifiSubscriptionCard: History page
+        <NotifiSubscriptionCard
+          darkMode
+          cardId=""
+          demoByJSONConfig={{ view: 'edit', data: JSON.parse(data) }}
+        />
+        Dummy Demo NotifiSubscriptionCard: Expired page
+        <NotifiSubscriptionCard
+          darkMode
+          cardId=""
+          demoByJSONConfig={{ view: 'expired', data: JSON.parse(data) }}
+        />
+        Dummy Demo NotifiSubscriptionCard: Verify page
+        <NotifiSubscriptionCard
+          darkMode
+          cardId=""
+          demoByJSONConfig={{ view: 'verify', data: JSON.parse(data) }}
+        />
+      </NotifiContext>
```

Comment:
ohhh, I now see why we wanted to structure things this way. This does feel pretty "clean" because at first sight the "demo config" modifies the subscription card.

However, I think lifting the demoByJSONConfig up to `NotifiContext` might be cleaner (since we don't want it to have any functionality when we're in demo mode)

Reply:
Very appreciate this hint, it is now moved up to `NotifiContext`